marc_to_solr/lib/traject_config.rb

```diff
@@ -895,7 +895,7 @@
   next if subjects.present?
   record.fields.select { |field| field.tag[0] == '6' }.each do |field|
     next unless field['2'] == 'fast'
-    value = field['a']
+    value = field['a'].delete('.')
```

Comment:
Is there any chance `field['a']` could return nil?
```suggestion
    value = field['a']&.delete('.')
```

Reply:
Oh, actually one more question sorry!  Wouldn't this remove periods within the subfield a, not just at the end?  I wonder if there are some fast headings that legitimately have "." inside them.

Reply:
yes it would remove them. I handled the period failure in orangelight so maybe I can close this PR?

Reply:
@sandbergja I will wait to discuss this with @mzelesky .

Reply:
@sandbergja I used delete_suffix to delete the . at the end of the string.